index.js

```diff
@@ -0,0 +1,60 @@
+const { Bot } = require('grammy');
+
+const bot = new Bot('5532232117:AAG_8EgKVkKd04srN7B5aHLKatC3biacIHc');
+bot.use((ctx, next) => {
+    ctx.state = {};
+    return next();
+})
+
+const middlewareMatchToSplit = function (ctx, next){
+    if(ctx.match === ''){
+        return ctx.reply("Please enter a number")
+    }
+    ctx.state.arguments = ctx.match.split(' ');
+    const validateNumbers = ctx.state.arguments.some((elem) => {
+        return Number.isNaN(Number(elem))
```

Comment:
A few lines below you're using + statement to convert value to number. Then why use Number here?

Reply:
Done